### DISCUSSION TITLE
Possibility to autoload nested plugin directories

### DISCUSSION THREAD

**Discussion by dapc11** [LazyVim/LazyVim#610] [Q&A]
Hello,
First of all, thanks for a great product.

I would like to split up my plugins into sections, like the structure on the LazyVim documentation.

Currently I have the following structure:

.
├── coding
│   ├── cmp.lua
│   ├── dap.lua
│   ├── git.lua
│   ├── lsp.lua
│   ├── lsp_servers
│   │   ├── gopls.lua
│   │   ├── jdtls.lua
│   │   ├── luals.lua
│   │   └── pyright.lua
│   ├── mason.lua
│   ├── neotest.lua
│   ├── null.lua
│   ├── project.lua
│   ├── pythonpath.lua
│   └── zk.lua
├── disable.lua
├── editor
│   ├── colorizer.lua
│   ├── dial.lua
│   ├── easyalign.lua
│   ├── luasnip.lua
│   └── surround.lua
**├── init.lua**
├── ui
│   ├── alpha.lua
│   ├── lualine.lua
│   ├── neotree.lua
│   ├── noice.lua
│   ├── scrollbar.lua
│   ├── telescope.lua
│   ├── theme.lua
│   ├── treesitter.lua
│   └── windows.lua

With the `init.lua` in `plugins/` requiring each and every entry.
Is it possible to have LazyVim automatically loading nested directories under the `plugins/` folder? This would remove the init.lua in my case and get me more structure, and the smooth loading that is there by default in LazyVim.

Thanks in advance!

**Comment by karimlevallois** [comment]
I love this structure, would you mind sharing your dotfiles?

**Reply by dapc11** [reply]
Hi, I've moved away from lazyim to my own config. But it is heavily influenced by lazyvim:

https://github.com/dapc11/dnvim
